src/main.rs

```diff
@@ -17,27 +17,103 @@
 
 extern crate rocksdb;
 extern crate test;
-use rocksdb::{Options, RocksDB, MergeOperands, new_bloom_filter, Writable};
+use rocksdb::{Options, RocksDB, MergeOperands, new_bloom_filter, Writable, };
```

Comment:
I just checked, it seems to require editing the code before you should run it...I think it will be great soon, but not today :)